airflow/hooks/dbapi.py

```diff
@@ -259,6 +266,27 @@ def get_cursor(self):
         """Returns a cursor"""
         return self.get_conn().cursor()
 
+    @staticmethod
+    def _split_sql_statements(sql, strip_comments=True, strip_semicolon=True, **kwargs):
+        """
+        Split multiple sql statements in string
+
+        :param sql: sql statements.
+        :type sql: str or list.
+        :return: splitted sql statements.
+        :rtype: list.
+        """
+        if isinstance(sql, str):
+            sql = [sql]
```

Comment:
I don’t think this is needed? Because you always wrap a str in a list on the call site (in `run`). Since this is a private method, always requiring a `List[str]` would make things cleaner. A type annotation on the function argument would help as well.